components/hero/hero.module.css

```diff
@@ -0,0 +1,42 @@
+/** Hero styles **/
+.hero{
+    margin-top:2rem;
```

Comment:
As mentioned in the Hero.jsx file keep a space inset from two throughout the files.